app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -26,7 +26,8 @@ const getAmiCharts = (chartsToLoad) => {
 const getAmis = async (chartsToLoad) => {
   const promises = map(chartsToLoad, chart => getAMIAction({ chartType: chart.ami_chart_type, chartYear: chart.ami_chart_year }))
   const amis = await Promise.all(promises)
-  return amis
+  // convert list of ami lists into one big array
```

Comment:
when amis is returned it's either
[[{}, {}, {}]] if there's only one kind of AMI chart, and [[{},{},{}], [{},{},{}], ...] if there are multiple AMI charts.

It might make sense to move this logic into where we actually find which AMI chart to use in the confirmed HH income file.